src/container/Footer/Footer.styles.ts

```diff
@@ -0,0 +1,9 @@
+import { Box, styled } from '@mui/material';
+
+export const StyledBox = styled(Box)(
+    ({ theme: { spacing, borderRadius } }) => ({
+        margin: 0,
+        padding: `${spacing(12)}  ${spacing(8)}`,
```

Comment:
```suggestion
        padding: spacing(12, 8),
```
**IMPORTANT**: Make sure that simple fixes like these are not repeated across PRs.